cylc/flow/cfgspec/workflow.py

```diff
@@ -802,9 +802,21 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     :ref:`SequentialTasks`.
             ''')
 
+        Conf('spawn from xtriggers sequentially', VDR.V_BOOLEAN, False,
```

Comment:
Not sure I like that as a config item name. I understand what it means, but it's long and we haven't described the concept in this way ("spawning *from* an xtrigger") before.  I think a short snappy name is fine, and we can define exactly what it means:
```suggestion
        Conf('sequential xtriggers', VDR.V_BOOLEAN, False,
```

Reply:
haha, I love how you critique your own suggestions!

Reply:
🤦‍♂️  In that case I prefer my NEW suggestion 😁